packages/publish-bundle/LICENSE

```diff
@@ -0,0 +1,201 @@
+                                 Apache License
```

Comment:
How did you scaffold this new project? Would you please leave some clues in the commit message?

Reply:
I’ll leave a note. It’s only semi automatic.